src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationUtilities.cs

```diff
@@ -147,8 +148,8 @@ internal static class SemanticClassificationUtilities
                 {
                     using var _ = ArrayBuilder<ClassifiedSpan>.GetInstance(out var classifiedSpans);
 
-                    await AddSemanticClassificationsAsync(
-                        document, snapshotSpan.Span.ToTextSpan(), classificationService, classifiedSpans, cancellationToken: cancellationToken).ConfigureAwait(false);
+                    await SemanticClassificationCacheUtilities.AddSemanticClassificationsAsync(
+                        document, snapshotSpan.Span.ToTextSpan(), classificationService, classifiedSpans, isRazorDoc: false, cancellationToken).ConfigureAwait(false);
```

Comment:
The `isRazorDoc` parameter was added for testing purposes so we can compare returning only semantic tokens vs. returning both semantic and syntactic tokens.